py/tests/release.py

```diff
@@ -0,0 +1,78 @@
+import os
+import getopt
+import sys
+import shutil
+import time
+import unittest
+
+import swagger_client
+from swagger_client.rest import ApiException
+from swagger_client.apis.external_api import ExternalApi
+from swagger_client.api_client import ApiClient
+
+def build_dir():
+    return "build_dir"
+
+def executable():
+    return os.path.join(build_dir(), "bin", "epoch")
+
+def extract_tarball(tarball_name):
+    extract_dir = build_dir()
```

Comment:
Please do not change the build dir. Travis uploads a file from the build dir (see `file_glob: true` and `file: _build/prod/rel/epoch/*-epoch-0.1.0.tar.gz` in .travis.yml) and extracting stuff in the build dir may confuse the upload by Travis. Please use a temporary folder for extracting tarball.

Reply:
BTW that file path in travis.yml shall be made more dynamic e.g. `_build/prod/rel/epoch/*-epoch-*.tar.gz` (untested).

Reply:
Yes, it should be